stdlib/ballerina-builtin/src/main/ballerina/auth/Package.md

```diff
@@ -4,21 +4,21 @@ This package provides a set of default authentication providers that can be exte
 
 ### Authentication Provider
 
-An authentication provider defines an authentication scheme that could be used to protect endpoints. The `auth:AuthProvider` type acts as the interface for all the authentication providers. Any type of implementation, such as LDAP, JDBC, and file based, should be object-wise similar. 
+An authentication provider defines an authentication scheme that could be used to protect endpoints. The `auth:AuthProvider` type acts as the interface for all the authentication providers. Any type of implementation, such as LDAP, JDBC, and file based, should be similar object-wise. 
 
-By default there are two implementations of that. They are, `auth:ConfigAuthProvider` which authenticates based on usernames and passwords stored in ‘ballerina.conf’, and `auth:JWTAuthProvider` which authenticates by validating a JWT. It is possible to implement more such authentication mechanisms.
+By default, there are two implementations of the `auth:AuthProvider`. They are, the `auth:ConfigAuthProvider`, which authenticates based on usernames and passwords stored in the `ballerina.conf` file, and the `auth:JWTAuthProvider`, which authenticates by validating a JWT. It is possible to implement more such authentication mechanisms.
 
 When creating a new authentication provider, there are two functions that need to be implemented. 
-- `authenticate` : authenticate the user based on a credential, which can be username/password or a token such as JWT.
-- `getScopes` : provide the scopes associated with the user. Scopes are primarily permissions that are required to access a protected resource. 
+- `authenticate` : Authenticate the user based on a credential, which can be username/password, or a token such as JWT.
+- `getScopes` : Provide the scopes associated with the user. Scopes are primarily permissions that are required to access a protected resource. 
 
 ### Config Auth Provider
 
 `ConfigAuthProvider` is an implementation of the `AuthProvider` interface, which uses the Ballerina configuration file
- to store usernames, passwords, scopes and the relevant associations.
+ to store usernames, passwords, scopes, and relevant associations.
```

Comment:
Can we change the following: "which uses the Ballerina configuration file" -> "which uses a Ballerina configuration file" (or just simply configuration file)

Reply:
fixed with commit https://github.com/ballerina-platform/ballerina-lang/pull/8263/commits/9de8db89d6e6b8b694d1ab18b4315271a8639995